### PR TITLE
CHK-1582-openapi-aca

### PR DESCRIPTION
**Pull request by Fernando-Granato**
Depends on https://github.com/pagopa/pagopa-infra/pull/1011

### List of changes
Added new apim configuration for aca.

New new base_policy file
New openapi.json file

### Motivation and context
Added the configuration for the new pagopa-aca API group.

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

**Review by pietro-tota — APPROVED** [review]
🚀 Just wait the dependent PR to be merge first before apply and then merge this one

File: src/domains/aca-app/api/aca/v1/_openapi.json.tpl [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+          "entityFullName",
+          "iuv",
+          "amount",
+          "description",
+          "expirationDate"
+        ],
+        "description": "Request body for creating a new transaction",
+        "properties": {
+          "paFiscalCode": {
+            "type": "string"
```
Comment (pietro-tota) [inline src/domains/aca-app/api/aca/v1/_openapi.json.tpl:146]: Add a brief description for each request fields

File: src/domains/aca-app/api/aca/v1/_openapi.json.tpl [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/NewDebtPositionRequest"
+              }
+            }
+          }
+        },
+        "responses": {
+          "201": {
```
Comment (pietro-tota) [inline src/domains/aca-app/api/aca/v1/_openapi.json.tpl:50]: Should here be also 200 status code used for debt position deletion/update requests?

File: src/domains/aca-app/api/aca/v1/_openapi.json.tpl (hunk 1 above)
Comment (Fernando-Granato) [reply src/domains/aca-app/api/aca/v1/_openapi.json.tpl:147]: Fixed with 17cec1bb7b114731d5cbfd78a517cd43a1abacbf 🙂

Comment (Fernando-Granato) [reply src/domains/aca-app/api/aca/v1/_openapi.json.tpl:147]: Fixed with be0ebd08a30e1463f6f7e0df0803d9d46b88ae25 🙂

File: src/domains/aca-app/api/aca/v1/_openapi.json.tpl (hunk 2 above)
Comment (Fernando-Granato) [reply src/domains/aca-app/api/aca/v1/_openapi.json.tpl]: Fixed with be0ebd08a30e1463f6f7e0df0803d9d46b88ae25 🙂

**Review by pietro-tota — APPROVED** [review]

**Review by infantesimone — APPROVED** [review]

**Pull request merged**